candle-core/src/metal_backend/device.rs

```diff
@@ -257,7 +323,8 @@ impl MetalDevice {
             return Ok(b.clone());
         }
 
-        let size = buf_size(size);
+        // Buffers on metal must be
```

Comment:
Incomplete comment, `must be powers of two`?

Reply:
(For context I went with power of two so that buffer reuse would be more likely)